MIAUI/README.md

```diff
@@ -0,0 +1,19 @@
+---
+name: .NET MAUI - MIAUI
+description: "This sample demonstrates how to implement authentication in a .NET MAUI to-do app"
+page_type: sample
+languages:
+- csharp
+- xaml
+products:
+- dotnet-maui
+- dotnet-core
+urlFragment: animations
+---
+
+# MIAUI
+
+This sample demonstrates how to implement authentication using Azure AD B2C in a .NET MAUI to-do app
+
```

Comment:
Awesome screenshots! Be sure to include them in the README here as well so that they can shine :)